classroom.rb

```diff
@@ -0,0 +1,14 @@
+attr_accessor :label
+attr_reader :students
+
+class Classroom
```

Comment:
- [ ] Please put your accessors methods  inside the `Classroom` class to access them inside your class

Reply:
Oh! Thanks, I did not notice it!

Reply:
You are welcome @CarlosE-Feliz